usaspending_api/etl/subaward_etl.py

```diff
@@ -180,9 +180,16 @@ def load_subawards(submission_attributes, db_cursor):
         recipient = LegalEntity.objects.filter(recipient_unique_id=row['duns']).first()
 
         if not recipient and row['duns']:
+            recipient_name = row['awardee_name']
+            if recipient_name is None:
+                recipient_name = row['awardee_or_recipient_legal']
+            if recipient_name is None:
+                logger.warn("Subaward number {} has no recipient name; skipping...".format(row['subaward_num']))
```

Comment:
Yes, that is correct. Currently, it will pick up a record where a DUNS matches, or, if there is a supplied duns and no match, it will create the recipient. Otherwise it leaves the recipient NULL on the subaward.